src/App.js

```diff
@@ -1,5 +1,68 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  // 전체 실행 흐름
+  async run() {
+    Console.print("덧셈할 문덧셈할 문자열을 입력해 주세요."); // 시작
+
+    const INPUT = await Console.readLineAsync(""); // 사용자 입력
```

Comment:
Console.readLineAsync()가 nodejs의 [rl.question](https://nodejs.org/api/readline.html#rlquestionquery-options-callback)을 이용한 함수여서 이렇게 변경해도 될 것 같아요! 👍
```suggestion
    const INPUT = await Console.readLineAsync("덧셈할 문자열을 입력해 주세요."); // 사용자 입력
```